tests/test_overfocus.py

```diff
@@ -60,23 +60,23 @@ def test_get_transformation_matrix(params):
                 'y_px': 3.,
                 'x_px': -7.,
             },
-            (3, -7)
+            (1.5, -3.5)
```

Comment:
Maybe we can re-add an "example" here that is the identity map? Would that be `camera_length=2`?

Reply:
Ah, there's one which uses the `detector_pixel_size` below...